test/NCI.OCPL.Api.CTSListingPages.Tests/Tests/Services/ESListingInfoQueryServiceTest.GetByIds.cs

```diff
@@ -0,0 +1,137 @@
+using System;
```

Comment:
Missing a test for the Elasticsearch connection failing. (e.g. returning only a partial response.)

Reply:
Included a test for this!